### ISSUE TITLE
Error & Feature Request: Make it vue-cli4 compatible

### ISSUE THREAD

**Issue by bsienn-khan** [GregYankovoy/vue-cli-plugin-bootstrap-vue#6]
Hi, Please make it compatible with vue-cli4, as it is throwing the following warning.
Vue-Cli ^4.1.1
Vue: ^2.6.11
Sass-loader: ^8.0.0 latest
```

🚀  Invoking generator for vue-cli-plugin-bootstrap-vue...
 WARN  conflicting versions for project dependency "sass-loader":

- ^8.0.0 injected by generator "undefined"
- ^7.1.0 injected by generator "vue-cli-plugin-bootstrap-vue"

Using newer version (^8.0.0), but this may cause build errors.
```

**Comment by GregYankovoy** [comment]
Hi.  Thanks for bringing this to my attention.  I hadn't realized v4 of the CLI had come out already.  The CLI plugin development documentation doesn't mention anything in particular in terms of migration from v3 to v4.  

For the moment, I've addressed the errors shown in your output by moving from node-sass to sass & updating sass-loader to major version 8.  

The updates should be available in vue-cli-plugin-bootstrap-vue 0.6.0.  Give it a try and let me know how it goes.

Thanks!

**Comment by bsienn-khan** [comment]
Hi, Thanks for the fix. It works fine now.

Here are some thoughts in my mind.

1) I see you updated  the readme file and renamed the vie-cli3  to only say vue-cli, which ofcource would represent any latest version of vu-cli, normally i guess.
As you alread decided to change the vue-cli3 to use vue-cli, I suggest you to change or add the title and tags, they stiil show vue-cli3.

2) I see you remove node-sass in favor of sass only. when I create a new project using cli, it asks me whether i need css pre processors support and gives 2 option, please see images bellow.

![1](https://user-images.githubusercontent.com/3149876/71515269-3f5d4680-28c4-11ea-8ad8-7dcaa1008342.png)
![2](https://user-images.githubusercontent.com/3149876/71515271-44ba9100-28c4-11ea-8d04-11617b9971cf.png)

I choose node-sass as required option.

I'm new to vue, so not sure removing node-sass and opting for sass as you did effects in any ways or not. Just wanted to let you know. and if you can enlighten me on this a bit that would be great.

Again, thanks for the quick fix. You may close the issue as fixed.
